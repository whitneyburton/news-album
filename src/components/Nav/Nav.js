import React from 'react';
import { NavLink } from 'react-router-dom';

export const Nav = () => {
  return (
    <div className="Nav--NavLink-container">
      <NavLink exact to="/" className="Nav--nav-btn" activeClassName="active">
        Home
      </NavLink>
      <NavLink
        to="/national-geographic"
        className="Nav--nav-btn"
        activeClassName="active"
      >
        National Geographic
      </NavLink>
      <NavLink
        to="/new-scientist"
        className="Nav--nav-btn"
        activeClassName="active"
      >
        The New Scientist
      </NavLink>
      <NavLink
        to="/crypto-coins"
        onClick={e => e.preventDefault()}
        className="Nav--nav-btn disabled"
      >
        Crypto Coins News
      </NavLink>
    </div>
  );
};
