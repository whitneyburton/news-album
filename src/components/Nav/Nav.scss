.Nav--NavLink-container {
  display: flex;
  flex-direction: row;
  justify-content: space-evenly;
  margin-bottom: 20px;

  .Nav--nav-btn {
    width: 30%;
    text-align: center;
    margin: auto 22px;
    text-decoration: none;
    color: #fff;
    font-weight: 300;
    letter-spacing: 2px;
    padding: 7px;
    border-radius: 12px;
    background-color: #4C3E3E;
  }
}