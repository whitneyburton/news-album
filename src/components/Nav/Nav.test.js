import React from 'react';
import { shallow } from 'enzyme';
import { Nav } from './Nav';

describe('Nav', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Nav />);
    expect(wrapper).toMatchSnapshot();
  });
});
