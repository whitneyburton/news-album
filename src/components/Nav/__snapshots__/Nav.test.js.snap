// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match the snapshot 1`] = `
<div
  className="Nav--NavLink-container"
>
  <NavLink
    activeClassName="active"
    aria-current="page"
    className="Nav--nav-btn"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="active"
    aria-current="page"
    className="Nav--nav-btn"
    to="/national-geographic"
  >
    National Geographic
  </NavLink>
  <NavLink
    activeClassName="active"
    aria-current="page"
    className="Nav--nav-btn"
    to="/new-scientist"
  >
    The New Scientist
  </NavLink>
  <NavLink
    activeClassName="active"
    aria-current="page"
    className="Nav--nav-btn"
    to="/crypto-coins"
  >
    Crypto Coins News
  </NavLink>
</div>
`;
